js/customizer.js

```diff
@@ -11,32 +11,26 @@
 	// Site title and description.
 	wp.customize( 'blogname', function( value ) {
 		value.bind( function( to ) {
-			$( '.site-title a' ).text( to );
-		} );
-	} );
+			document.querySelector( '.site-title a' ).textContent( to );
+		});
+	});
 	wp.customize( 'blogdescription', function( value ) {
 		value.bind( function( to ) {
-			$( '.site-description' ).text( to );
-		} );
-	} );
+			document.querySelector( '.site-description' ).textContent( to );
+		});
+	});
 
 	// Header text color.
 	wp.customize( 'header_textcolor', function( value ) {
 		value.bind( function( to ) {
 			if ( 'blank' === to ) {
-				$( '.site-title, .site-description' ).css( {
-					'clip': 'rect(1px, 1px, 1px, 1px)',
-					'position': 'absolute'
-				} );
+				document.querySelector( '.site-title, .site-description' ).style.clip = 'auto';
```

Comment:
If you mean there are multiple `.site-title`  and `.site-description` you can use `Object.assign`. I haven't used it before, always could solve it easy with just `document.querySelector`.

Reply:
There are multiple statements that start `document.querySelector( '.site-title, .site-description' ).style` to change more than one property of the element's `style`. Using `Object.assign` like this would mean that `document.querySelector` is only called once.